<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertDialog</class>
 <widget class="QDialog" name="InsertDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>insert by position</string>
  </property>
  <widget class="QPushButton" name="button_apply">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_cancel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_at_character">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>insert at character:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_insert_what">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>insert what:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_old">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>old:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_new">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>new:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_old">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_new">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>line_1</tabstop>
  <tabstop>line_2</tabstop>
  <tabstop>button_apply</tabstop>
  <tabstop>button_cancel</tabstop>
  <tabstop>line_old</tabstop>
  <tabstop>line_new</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
